cylc/flow/parsec/fileparse.py

```diff
@@ -239,27 +268,63 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
         flines = inline(
             flines, fdir, fpath, False, viewcfg=viewcfg, for_edit=asedit)
 
+    template_vars['CYLC_VERSION'] = __version__
+
     # process with EmPy
     if do_empy:
+        if (
+            extra_vars['templating detected'] == 'empy:suite.rc' and
```

Comment:
Just return the name of the template engine without the `:suite.rc` suffix.

```suggestion
            extra_vars['templating detected'] == 'empy' and
```